src/utils/helpers/structure/generateReviewProgress.ts

```diff
@@ -12,18 +12,16 @@ const generateReviewProgress = (newStructure: FullStructure) => {
 const generateSectionReviewProgress = (section: SectionState) => {
   section.reviewProgress = getSums(Object.values(section.pages))
 }
-// Helper to see if thisReviewLatestResponse is linked to latest application response
-const isLatestReviewResponseUpToDate = (element: PageElement) =>
-  element.response?.id === element.thisReviewLatestResponse?.applicationResponse?.id
 
 const generatePageReviewProgress = (page: Page) => {
   const totalReviewable = page.state.filter(
-    (element) => element.isAssigned && element?.element.isVisible
+    (element) =>
+      element.isAssigned && element?.element.isVisible && element?.latestApplicationResponse?.id
   )
 
   // Only consider review responses that are linked to latest application response
   const totalReviewableLinkedToLatestApplicationResponse = totalReviewable.filter(
-    isLatestReviewResponseUpToDate
+    ({ isPendingReview }) => !isPendingReview
   )
 
   const doneConform = totalReviewableLinkedToLatestApplicationResponse.filter(
```

Comment:
Hmm, I am not exactly sure what you mean by this comment: `why is totalReviewableLinkedToLatestApplicationResponse is == doneConform`

But the need fro `totalReviewableLinkedToLatestApplicationResponse` is to differentiate between 'active' current review response and the ones done in the past that are not active anymore since new application response was written